utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/ConfluentSchemaRegistryClient.scala

```diff
@@ -59,6 +62,22 @@ class DefaultConfluentSchemaRegistryClient(override val client: CSchemaRegistryC
       client.getAllVersions(subject).asScala.toList
     }
 
+  override def getSchemaById(id: Int): SchemaWithMetadata = {
+    //todo schemaCache is subject-version keyed, how to get it by id
```

Comment:
what do we want to do here?

Reply:
it was meant for cached version - it's not obvious how to cache this by id - but for now I removed todo